src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -546,6 +553,9 @@ where
                 // right now.
                 let mut emit_ts = Timestamp::from((*now_fn)());
 
+                // The last frontier we compacted the remap trace to, starting at [0].
+                let mut last_compaction_since = Antichain::from_elem(Timestamp::default());
```

Comment:
I think `::minimum()` is the correct thing to call here. I haven't seen an instance where it differs from the default but at the same time Frank didn't use the `Default` trait for it for some reason so let's stick with the convention